packages/schemas/src/player/stats.ts

```diff
@@ -106,6 +110,7 @@ export class PlayerStats {
     this.paintball = new Paintball(data?.stats?.Paintball ?? {});
     this.parkour = new Parkour(data?.parkourCompletions ?? {});
     this.quake = new Quake(data?.stats?.Quake ?? {}, data?.achievements ?? {});
+    this.seasonal = new Seasonal(data?.stats?.Arcade ?? {});
```

Comment:
Make this part of the `Arcade` class not its own field in the `PlayerStats` class